<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WishPage</class>
 <widget class="QWidget" name="WishPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>597</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">QLabel{
  font-family:&quot;HarmonyOS Sans SC&quot;;
  color : #666666
}</string>
       </property>
       <property name="text">
        <string>角色祈愿</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QTextEdit" name="urlTextEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="showWishDataButton">
       <property name="styleSheet">
        <string notr="true">QPushButton{
  font-family:&quot;HarmonyOS Sans SC&quot;;
  color : #666666
}</string>
       </property>
       <property name="text">
        <string>获取祈愿信息</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="styleSheet">
      <string notr="true">QScrollBar:vertical
{
  width: 4px;
  border:0px solid;
  border-radius: 2px;
  margin: 0px,0px,0px,0px;
  background-color: transparent;
  background:#F5F5F5;
}
 
QScrollBar:vertical:hover
{
  width: 4px;
  border:0px solid;
  margin: 0px,0px,0px,0px;
  background:#F5F5F5;
}
 
QScrollBar::handle:vertical
{
  width:4px;
  background:#E1E1E1;
  border-radius: 2px;
  height: 40px;
  
}
 
QScrollBar::handle:vertical:hover
{
  background:#E1E1E1;
  border-radius: 2px;
}
 
QScrollBar::add-line:vertical
{
  height:11px;
  background-color: transparent;
  subcontrol-position:bottom;
  border:0px solid;
  border-radius: 2px;
 
}
QScrollBar::sub-line:vertical
{
  height:11px;
  background-color: transparent;
  subcontrol-position:top;
  border:0px solid;
  border-radius: 2px;
}
 
QScrollBar::add-page:vertical
{
  background-color: #F5F5F5;
  border:0px solid;
  border-radius: 2px;
}
 
QScrollBar::sub-page:vertical
{
  background-color: #F5F5F5;
  border:0px solid;
  border-radius: 2px;
}
 
QScrollBar::up-arrow:vertical{
  border:0px solid;
  border-radius: 3px;
}
 
QScrollBar::down-arrow:vertical {
  border:0px solid;
  border-radius: 3px;
}</string>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>595</width>
        <height>406</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
